frontend/src/components/Metrics/MetricsStep/DeploymentFrequencySettings/SingleSelection/style.tsx

```diff
@@ -1,8 +1,14 @@
 import { styled } from '@mui/material/styles'
-import { FormControl } from '@mui/material'
+import { Avatar, FormControl } from '@mui/material'
 
 export const FormControlWrapper = styled(FormControl)({
   margin: '0 0 2rem 2.5%',
   width: '95%',
   height: '2.5rem',
 })
+
+export const StyledAvatar = styled(Avatar)({
```

Comment:
duplicated code, ref to frontend/src/components/Common/ReportForThreeColumns/style.tsx